admin/css/debug-toolbar/toolbar.scss

```
@@ -510,3 +522,47 @@
 .debug-bar-noverflow {
     overflow: hidden;
 }
+
+.debug-bar-dtableRow{
+    display: table-row;
+  }
+  
+.debug-bar-dinlineBlock{
+    display: inline-block;
+}
+  
+.debug-bar-pointer{
+    cursor: pointer;
+}
+  
+.debug-bar-mleft4{
+    margin-left: 4px;
+}
+  
+.level-0{
```

```suggestion
.debug-bar-level-0{
```
Please add a  scope.